newIDE/app/src/AssetStore/PrivateGameTemplates/PrivateGameTemplateInformationPage.js

```diff
@@ -146,6 +150,111 @@ const PrivateGameTemplateInformationPage = ({
       gameTemplate => gameTemplate.id === privateGameTemplateListingData.id
     );
 
+  const templatesIncludedInBundleTiles = React.useMemo(
```

Comment:
These two inlines useMemo (this one, and the next one) are quite "heavy" to read when you go through the component. I think they would benefit from being into two functions (that could almost be shared between asset packs and templates?), so that the component stays simple?

Reply:
Something like `getTemplatesIncludedInBundle`. The TILES themselves are fine to put in the component, but the logic to get them could be outside.

Reply:
I've moved the 3 memos that were in common to a separate helper, much cleaner 👍